### PR TITLE
Use Semgrep App instead of Semgrep Community and Semgrep Team (where appropriate)

### PR DESCRIPTION
**Pull request by pabloest**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ Semgrep CI makes network requests in accordance with the data storage mentioned
 # Configuration
 
 #### How do I configure Semgrep for different projects?
-Semgrep Community and Team editions provide centralized policy management. See [Managing CI policy](managing-policy.md) for more details.
+Semgrep App’s Community and Team tiers provide centralized policy management. See [Managing CI policy](managing-policy.md) for more details.
```
Comment (dlukeomalley) [inline docs/faq.md:130]: Is it necessary to mention Community and Team here or could this be shortened to "Semgrep App provides centralized"? We could go further and reference the specific feature: Semgrep App's Dashboard

File: docs/integrations.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Semgrep integrates into the development flow end-to-end, from code conception in
 
 # Continuous integration (CI)
 
-The following instructions use [Semgrep CI](https://github.com/returntocorp/semgrep-action) and require a free [Semgrep Community](https://semgrep.dev/manage) or paid Semgrep Team account. `SEMGREP_DEPLOYMENT_ID` and `SEMGREP_APP_TOKEN` information is available at [Manage > Settings](https://semgrep.dev/manage/settings) after login.
+The following instructions use [Semgrep CI](https://github.com/returntocorp/semgrep-action) and require a [Semgrep App](https://semgrep.dev/manage) account (free or paid tier). `SEMGREP_DEPLOYMENT_ID` and `SEMGREP_APP_TOKEN` information is available at [Manage > Settings](https://semgrep.dev/manage/settings) after login.
```
Comment (dlukeomalley) [inline docs/integrations.md:9]: Throughout the docs we use the language "free or paid tier" frequently. Do you feel the reminder is necessary?

File: docs/managing-policy.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Managing CI policy
 
-Semgrep Community and Semgrep Team provide infrastructure for managing Semgrep across many projects. Create policies from their web UI and apply those policies to projects in a many-to-many mapping. A policy is a simple collection of rules and a definition of what to do with rule results: fail the Semgrep CI run and/or send non-blocking notifications to third-party services like Slack.
+Semgrep App provides infrastructure for managing Semgrep across many projects. Create policies from their web UI and apply those policies to projects in a many-to-many mapping. A policy is a simple collection of rules and a definition of what to do with rule results: fail the Semgrep CI run and/or send non-blocking notifications to third-party services like Slack.
```
Comment (dlukeomalley) [inline docs/managing-policy.md:3]:
```suggestion
Semgrep App provides infrastructure for managing Semgrep across many projects. Create policies from its web UI and apply those policies to projects in a many-to-many mapping. A policy is a simple collection of rules and a definition of what to do with rule results: fail the Semgrep CI run and/or send non-blocking notifications to third-party services like Slack.
```

**Review by dlukeomalley — APPROVED** [review]
Overall lgtm, with some questions on language use for the future.

File: docs/faq.md (hunk 1 above)
Comment (pabloest) [reply docs/faq.md:130]: I’m fine with that change. I was looking to find _some_ place to introduce the tiers, e.g., to inform the user that they will need to sign up for something in order to manage Semgrep across projects. I don’t have strong feelings though.

File: docs/integrations.md (hunk 2 above)
Comment (pabloest) [reply docs/integrations.md:9]: Probably not necessary to use it quite so frequently.

Comment (pabloest) [reply docs/integrations.md:9]: @dlukeomalley ugh how do I accept your suggestion here?

**Pull request merged**